packages/core/src/components/WarningPanel/WarningPanel.tsx

```diff
@@ -47,34 +48,35 @@ const styles = theme => ({
   messageText: {
     color: theme.palette.warningText,
   },
-});
+}));
 
 /**
  * WarningPanel. Show a user friendly error message to a user similar to ErrorPanel except that the warning panel
  * only shows the warning message to the user
  */
-class WarningPanel extends Component {
-  static propTypes = {
-    message: PropTypes.node.isRequired,
-  };
 
-  render() {
-    const { classes, title, message, children } = this.props;
-    return (
-      <div className={classes.message}>
-        <div className={classes.header}>
-          <ErrorOutlineStyled />
-          <Typography className={classes.headerText} variant="subtitle1">
-            {title}
-          </Typography>
-        </div>
-        {message && (
-          <Typography className={classes.messageText}>{message}</Typography>
-        )}
-        {children}
+type Props = CSS.Properties & {
```

Comment:
How come you have `CSS.Properties` here?

Reply:
Fixed